python/lsst/ts/wep/task/calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,1964 @@
+    )
+    donutQualityTable = connectionTypes.Output(
+        doc="Quality information for donuts",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="AstropyQTable",
+        name="donutQualityTable",
+    )
+
+
+class CalcZernikesNeuralTaskConfig(
+    pipeBase.PipelineTaskConfig,
+    pipelineConnections=CalcZernikesNeuralTaskConnections,  # type: ignore
+):
+    """Configuration for CalcZernikesNeuralTask.
+
+    Attributes
+    ----------
+    wavenetPath : str or None
+        Model Weights Path for wavenet. If None, TARTS will create a new
+        model with random weights (useful for testing).
+    alignetPath : str or None
+        Model Weights Path for alignet. If None, TARTS will create a new
+        model with random weights (useful for testing).
+    aggregatornetPath : str or None
+        Model Weights Path for aggregatornet. If None, TARTS will create a new
+        model with random weights (useful for testing).
+    datasetParamPath : str
+        Path to TARTS dataset parameters YAML file containing normalization
+        scaling factors, image processing parameters (CROP_SIZE, deg_per_pix,
+        mm_pix), model hyperparameters, and training data file paths
+    device : str
+        Device to use for calculations
+    nollIndices : list[int]
+        List of Noll indices to calculate. Default is Z4-Z23 (4-23),
+        excluding piston (Z1), tip (Z2), and tilt (Z3) which are
+        typically not measured in wavefront sensing.
+    cropSize : int
+        Size of donut crop in pixels (width and height). Default is 160 pixels,
+        which matches the TARTS neural network training data format.
+    """
+
+    wavenetPath: pexConfig.Field = pexConfig.Field(
+        doc="Model Weights Path for wavenet",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    alignetPath: pexConfig.Field = pexConfig.Field(
+        doc="Model Weights Path for alignet",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    aggregatornetPath: pexConfig.Field = pexConfig.Field(
+        doc="Model Weights Path for aggregatornet",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    datasetParamPath: pexConfig.Field = pexConfig.Field(
+        doc="Path to TARTS dataset parameters YAML file containing normalization "
+            "scaling factors, image processing parameters (CROP_SIZE, deg_per_pix, "
+            "mm_pix), model hyperparameters, and training data file paths",
+        dtype=str
+    )
+    device: pexConfig.Field = pexConfig.Field(
+        doc="Device to use for calculations",
+        dtype=str,
+        default="cuda"
+    )
+    nollIndices: pexConfig.ListField = pexConfig.ListField(
+        doc="List of Noll indices to calculate. Default is Z4-Z23 (4-23), "
+            "excluding piston (Z1), tip (Z2), and tilt (Z3) which are "
+            "typically not measured in wavefront sensing.",
+        dtype=int,
```

Comment:
Is this dtype correct given that you are passing a list? Have you tested passing a different list as input to check that I will not complain?